app/models/concerns/indexable.rb

```diff
@@ -140,3 +141,4 @@ def _geoloc
                  .map { |organisation| { lat: organisation.geolocation.x.to_f, lng: organisation.geolocation.y.to_f } }
   end
 end
+# rubocop:enable Metrics/BlockLength
```

Comment:
You don't need this if it applies to the whole file. Even better, you can also put the `rubocop:disable` comment right next to the beginning of the block it applies to, i.e.:

```ruby
module Indexable # rubocop:disable Metrics/BlockLength
  # ...
end
```

Reply:
or maybe exclude this file from this rule in rubocop.yml file